### PR TITLE
Add additional info to setup in Windows

### PR DESCRIPTION
**Pull request by daveres**
Add cpu and virtualization info to setup in Windows

### CODE REVIEW

**Comment by jhawkesworth** [comment]
Thanks for this, A few thoughts.

It would be nice if it also supported VirtualBox
on my test vm win32_bios's Version property is 'VBOX   -1'

I think the name 'ansible_product_name' is confusing for the attribute.  I know it needs to start 'ansible_' but perhaps 'ansible_virtualization_platform' or 'ansible_virtualization_vendor' 

I think I'd prefer it if the key was always present, but set to something like 'not detected' if it couldn't be detected.  Then you don't have to handle it not necessarily being present in your playbook logic.

I'd be interested to know what your use case is for this change is too.

**Comment by gregdek** [comment]
Thanks @daveres for this PR. This module is maintained by the Ansible core team, so it can take a while for patches to be reviewed. Thanks for your patience.

**Comment by bcoca** [comment]
You can assign a null/none/empty string/list/dict to the key value instead of creating an arbitrary value of 'not found' as the former is testable in general as a 'false' value in Ansible.

File: windows/setup.ps1 [hunk 1]
```diff
@@ -28,6 +28,29 @@ $result = New-Object psobject @{
...
+}
+if (((Get-WmiObject win32_bios).SerialNumber).tolower().contains("vmware")) {
+        $virtualization_type = "VMWare"
+}
+if (((Get-WmiObject win32_bios).Version).tolower().contains("xen")) {
+        $virtualization_type = "Xen"
+}
+if (((Get-WmiObject win32_bios).Version).tolower().contains("vbox")) {
+        $virtualization_type = "VirtualBox"
+}
```
Comment (nitzmahone) [inline windows/setup.ps1:44]: This isn't quite robust enough - have a look at https://blogs.technet.microsoft.com/heyscriptingguy/2015/03/30/use-powershell-to-detect-if-hypervisor-is-present/ and http://blogs.splunk.com/2013/08/12/detecting-your-hypervisor-from-within-a-windows-guest-os/ as starting points. I can see a number of cases where this code will false-trigger Hyper-V for real BIOSes. Need to figure out if we're under a Hypervisor at all first, then try to identify...

**Comment by gregdek** [comment]
@daveres A friendly reminder: this pull request has been marked as needing your action. If you still believe that this PR applies, and you intend to address the issues with this PR, just let us know in the PR itself and we will keep it open pending your changes.

[This message brought to you by your friendly Ansibull-bot.]

**Comment by gregdek** [comment]
@daveres Another friendly reminder: this pull request has been marked as needing your action. If you still believe that this PR applies, and you intend to address the issues with this PR, just let us know in the PR itself and we will keep it open. If you have addressed the issues and believe it's ready for review, please comment with the text "ready_for_review". If we don't hear from you within another 14 days, we will close this pull request.

[This message brought to you by your friendly Ansibull-bot.]

**Comment by gregdek** [comment]
@daveres A friendly reminder: this pull request has been marked as needing your action. If you still believe that this PR applies, and you intend to address the issues with this PR, just let us know in the PR itself and we will keep it open pending your changes. When you do address the issues, please respond with ready_for_review in your comment, so that we can notify the maintainer.

[This message brought to you by your friendly Ansibull-bot.]

**Comment by gregdek** [comment]
@daveres Another friendly reminder: this pull request has been marked as needing your action. If you still believe that this PR applies, and you intend to address the issues with this PR, just let us know in the PR itself and we will keep it open. If you have addressed the issues and believe it's ready for review, please comment with the text "ready_for_review". If we don't hear from you within another 14 days, we will close this pull request.

[This message brought to you by your friendly Ansibull-bot.]

**Comment by nitzmahone** [comment]
closing for inactivity

**Pull request closed without merging**